projects/2019_12_14_pkt-whitepaper-website.md

```diff
@@ -372,3 +372,589 @@ The applicant understands that the Network Steward is not a legal entity and no
+This plan is obviously very ambitious, but like ARPANET started as a couple of computers connected over the telephone PKT has begun this way as well.
+PKT also carries forward the belief that all communications should be encrypted, no matter what. This means, nobody should have to apply for https certificates, there should be no question of the origins of a data packet when it is received, and nobody should ever need a VPN to get another Internet connection when they already have one.
+If you believe that the internet should be owned by the people, and that no one should ever be denied access while bandwidth sits unused, if you want to look back on the days of $50 a month the way we look back on the days of $18 phone calls, come build with us. We're just getting started.
+Button: PKT Roadmap
+Button: PKT Origin Story
+
+#PKT Roadmap 
+There are four distinct stages of evolution which the PKT project will undergo. These stages may occur out of order at times, but the project will be viable in every state, just as a caterpillar is a viable creature before it becomes a butterfly.
+
+##The PKT Roadmap is:
+Build a VPN market where anyone can participate and the minimum price is free
+Support low cost Internet sharing hardware that allows people to earn PKT using their extra bandwidth
+Create a bandwidth market where there are virtual ISPs to do the network engineering so that building a mesh network is easy
+Support high performance networking hardware which automatically sells its bandwidth into the decentralized bandwidth market
+
+##Stage One, VPN Marketplace
+  
+Price will be decided by auction with a minimum bid of zero. In order to stay decentralized and achieve low enough cost of transactions to enable true micropayments, VPNs will be purchased using PKT, a cryptocurrency which is mined by performing encryption operations and expending Internet bandwidth. Anyone who wants to use a VPN, create their own, or mine PKT will easily be able to do so.
+
+##Stage Two, Support Internet Sharing
+  
+Once there are approximately one million installations of the VPN client application, PKT will begin supporting third party operators who offer internet sharing technology and hardware for home or business users. This technology and hardware will allow people to earn PKT with their unused internet bandwidth and like the VPN market, it will have a minimum bid of zero. Since it will be the VPN client application that connects to the Internet sharing hardware, there will be no privacy risk nor legal risk because all data will be sent through the VPN.
+
+##Stage Three, Tokenized Bandwidth Leases
+
+By tokenizing bandwidth leases Internet sharing hardware will be able to connect to each other and virtual ISPs will be able to observe the network, buy, sell and trade, relevant leases, and plan paths through it. This will make local mesh networking easy by moving the network engineering component to the virtual ISP layer.
```

Comment:
>This will make local mesh networking easy by moving the network engineering component to the virtual ISP layer.

It's not clear to me what this means. Is it that there will be presence at IXPs where people can buy leases? Better make this explicit.

Reply:
Your reaction reveals a significant issue with this description, and one I'm not sure how to remedy without going into great detail. I'm responsible for this one and what I was trying to say is that with bandwidth leasing, every router *is* an IXP, a virtual meet-me room where virtual ISPs are able to connect their bandwidth leases together. I feel like this is a big deal if people don't understand this point so please let me propose a couple of alternate wordings...

Reply:
What about:

> The tokenization of bandwidth leases will not only finance local mesh networks, but it will allow the emergence of *virtual ISPs*, organizations who build networks by knitting together bandwidth leases. When tens or hundreds of different virtual networks can coexist on the same routing device, every router becomes a mini-datacenter where virtual ISPs interconnect with one another.